generator/generic/static/generic_utilities.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module Inferno
+  module GenericUtilities
```

Comment:
We should give this a name that makes more sense when it's included in a sequence. `SequenceUtilities`? `SequenceHelpers`?

Having it located in `static` is a problem because it gets copied into each generated module, but this file should only exist and be loaded once. If it's going to exist with each module, then it needs to be namespaced.

Reply:
Renamed it to `GenericUtilities` and moved it into `lib/utils`.  Sequence base requires the file, but the individual sequences are responsible for including it.